.github/PULL_REQUEST_TEMPLATE/default.md

```diff
@@ -0,0 +1,46 @@
+<!-- 🎉🎉🎉 Thank you for the PR!!! 🎉🎉🎉 -->
+
+# Changes
+
+<!-- Describe your changes here- ideally you can get that description straight from
+your descriptive commit message(s)! -->
+
+# Submitter Checklist
+
+These are the criteria that every PR should meet, please check them off as you
+review them:
+
+- [ ] Includes tests for code changes, new features etc.
+- [ ] Includes documentation
+- [ ] Commit messages follow commit message best practices
+- [ ] Release notes block has been filled out
+
+# Release Notes
+
+<!--
+Describe any user facing changes here, or delete this block.
+
+Examples of user facing changes:
+- API changes
+- Bug fixes
+- Any changes in behavior
+- Changes requiring upgrade notices or deprecation warnings
+
+For pull requests with a release note:
+
+```release-note
```

Comment:
if we don't have a release notes bot hooked up, this feels deceptive

but i guess it can't hurt (can always be added after the fact)

Reply:
oh, we do!

is that always desirable? e.g. for like demo code